src/connection/Form.ts

```diff
@@ -91,14 +91,34 @@ export default class Form {
 
       this.items.forEach((fields, itemIndex) => {
          if (fields.length !== this.reportedFields.length) {
-            throw new InvalidParameterError(`Item ${itemIndex} does not contain all "reported" fields.`);
+             try{
+                 this.reportedFields.forEach((field, index) => {
```

Comment:
In which case is the server returning more fields than reported fields?

Reply:
you are right ... maybe it was to late when i wrote the code...